src/smart-components/group/add-group/users-list.js

```diff
@@ -57,8 +57,8 @@ const UsersList = ({ users, fetchUsers, isLoading, pagination, selectedUsers, se
     request={ fetchUsers }
     checkedRows={ selectedUsers }
     setCheckedItems={ setCheckedItems }
-    titlePlural="users"
-    titleSingular="user"
+    titlePlural="usernames"
```

Comment:
This is not correct. This will also change empty state, if you wanted to fix the filter placeholder that is being addressed in https://github.com/RedHatInsights/insights-rbac-ui/pull/166